heapster-saw/src/Verifier/SAW/Heapster/TypedCrucible.hs

```diff
@@ -4047,14 +4074,16 @@ visitEntry names can_widen blk entry =
 -- | Visit a block by visiting all its entrypoints
 visitBlock ::
   (PermCheckExtC ext, KnownRepr ExtRepr ext) =>
-  TypedBlock TCPhase ext blocks tops ret args ->
+  Bool -> TypedBlock TCPhase ext blocks tops ret args ->
```

Comment:
```suggestion
  Bool {- ^ can widen -} -> TypedBlock TCPhase ext blocks tops ret args ->
```

The documentation mentions a *can widen* parameter, and especially with this being a generic `Bool` it could be nice to be explicit in the haddocks.

Reply:
Yes, good suggestion here as well. I never taught myself to add Haddocks to arguments, but it's something I need to learn...